documentation/dev/src/explore/design/ledger/pos-integration.md

```diff
@@ -53,48 +49,49 @@ For slashing tokens, we implement a [PoS slash pool account](vp.md#pos-slash-poo
 
 The validator transactions are assumed to be applied with an account address `validator_address`.
 
-- `become_validator(consensus_key, staking_reward_address)`:
+- `become_validator(consensus_key, commission_rate, max_commission_rate_change)`:
```

Comment:
How precise do we care to be here with the input arguments? Technically, this function also requires the validator address and the current epoch as inputs at the moment.

Reply:
For current epoch, we can maybe extend the line 50 with e.g. `and assuming a transaction uses host environment function to find the current epoch`. I think we could also specify them in args to be more complete